### DISCUSSION TITLE
Help Overlay - Press F1 to bring up Hotkeys

### DISCUSSION THREAD

**Discussion by ChrisTitusTech** [hyprwm/Hyprland#1367] [General]
AwesomeWM has something like this <https://awesomewm.org/doc/api/libraries/awful.hotkeys_popup.widget.html>

Its a quick widget popup showing all the keys. What would be the best way to add this to Hyprland? The quick and dirty was is to use notify-send, but I want something more elegant and to fill most of the screen. 

Any ideas?

**Comment by vaxerski** [comment]
opening a top/overlay layer with eww is my first thought.

@fufexan any better ideas as the eww connoisseur?

**Comment by fufexan** [comment]
Eww *can* be used to show them. It would maybe take some time to create the layout though.
